RevenueCat/Scripts/Purchases.cs

```diff
@@ -390,7 +423,7 @@ public Package(PackageResponse response)
         {
             Identifier = response.identifier;
             PackageType = response.packageType;
-            Product = response.product;
+            Product = new Product(response.product);
```

Comment:
was this a bug?

Reply:
no, it's because before `response.product` was a `Product`, now it's a `ProductResponse`